draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -500,7 +502,7 @@ Examples of the transitions between phases are provided in <xref target="Example
 	     the current_rtt is greater than or equal to
              (saved_rtt / 2) or the current_rtt is less than or equal to (saved_rtt x 10)
```

Comment:
With this revision, there's confusion around the congestion_window_limited trigger, which seems now to be defined as a trigger for 2 phases. The text could first clarify it is a trigger for moving to Unvalidated phase:

If the sender is rate-limited <xref target="RFC7661"></xref>, it might send insufficient data to be able to validate transmission at the higher rate. Careful Resume is allowed to remain in the Reconnaissance Phase and to not transition to the Unvalidated Phase until the sender has more data ready to send in the transmission buffer
than is permitted by the current CWND (see trigger congestion_window_limited in <xref target="sec-QLOG").

Reply:
Unsure if this is now resolved?

Reply:
It's no longer confusing, but you could still reference the trigger in the Qlog here if you wanted!